src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  const partOfResult = { ...state };
+
+  for (const act of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
Shortening code is good - shortening variable names is bad

Reply:
+